api/window/src/test/java/quarks/test/window/WindowTest.java

```diff
@@ -325,6 +320,7 @@ public void countBatchWindowTest(){
                         tuple -> 0,
                         () -> new ArrayList<Integer>());
         window.registerPartitionProcessor(new BiConsumer<List<Integer>, Integer>() {
+            @SuppressWarnings("unused")
```

Comment:
Stupid question, why have a count variable, not use it and then mark it not used?

Why not just remove it?